compose/runtime/runtime/src/jvmMain/kotlin/androidx/compose/runtime/ActualJvm.jvm.kt

```diff
@@ -30,7 +30,7 @@ internal actual fun getCurrentThreadId(): Long = Thread.currentThread().id
 @InternalComposeApi
 actual fun identityHashCode(instance: Any?): Int = System.identityHashCode(instance)
 
-internal actual typealias TestOnly = org.jetbrains.annotations.TestOnly
+actual typealias TestOnly = org.jetbrains.annotations.TestOnly
```

Comment:
I see that there is one more place like this:
https://github.com/JetBrains/compose-multiplatform-core/blob/d9aec78d04bf59f82c5ae82ce0f662e3f9b7b9c9/compose/material3/material3-window-size-class/src/jvmMain/kotlin/androidx/compose/material3/windowsizeclass/TestOnly.jvm.kt#L19

Should it be updated too?

Reply:
I will try to remote it too. And use`TestOnly` annotation from the `runtime` module

Reply:
It is because we don't build native in this module yet